File: push/src/main/java/com/xiaomi/push/service/awake/module/IProcessData.java
package com.xiaomi.push.service.awake.module;

import android.content.Context;
import java.util.HashMap;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/push/service/awake/module/IProcessData.class */
public interface IProcessData {
    void sendByTinyData(Context context, HashMap<String, String> map);

    void sendDirectly(Context context, HashMap<String, String> map);

    void shouldDoLast(Context context, HashMap<String, String> map);
}

File: legacy/src/main/java/com/xiaomi/push/service/awake/module/IProcessData.kt
package com.xiaomi.push.service.awake.module
import io.github.magisk317.mipush.protocol.model.*

import android.content.Context
import java.util.HashMap

interface IProcessData {
    fun sendByTinyData(context: Context, map: HashMap<String, String>)

    fun sendDirectly(context: Context, map: HashMap<String, String>)

    fun shouldDoLast(context: Context, map: HashMap<String, String>)
}
